kv1/models.py

```diff
@@ -34,9 +38,9 @@ class Kv1Stop(models.Model):
     location = models.PointField()
 
     # Custom manager for geomodels/searches
-    objects = models.GeoManager()
+    objects = GeoManager()
```

Comment:
Volgens mij mag dit gewoon weg nu - is voor geo searches, maar dat doen we zo te zien niet.